<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="TYSS-Gali-Sai-Vasavi" failures="1" tests="1" name="Test3" time="7.376" errors="0" timestamp="2023-12-19T09:59:20 IST">
  <testcase classname="practice.cross2" name="login" time="7.354">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at practice.cross2.login(cross2.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Test3 -->
